src/ui/RichText.tsx

```diff
@@ -129,7 +137,7 @@ export type RichTextProps<Blocks extends Schema> = {
   text?: ComponentType<{children: string}>
 } & {
   [K in keyof typeof RichTextElements]?:
-    | ComponentType<JSX.IntrinsicElements[K]>
+    | ComponentType<JSX.IntrinsicElements>
```

Comment:
We still need to use the original type here: | ComponentType<JSX.IntrinsicElements[K]>

Reply:
Change has been reverted.